client/package.json

```diff
@@ -56,10 +57,9 @@
     "react-icons-kit": "^1.2.1",
     "reason-apollo": "^0.13.0",
     "reason-react": ">=0.4.0",
+    "static-server": "^2.2.1",
     "subscriptions-transport-ws": "^0.9.14",
-    "victory": "^32.0.0"
-  },
-  "devDependencies": {
+    "victory": "^32.0.0",
```

Comment:
Why is this a devDevependency?

Reply:
It's not, it's the **dependencies**

Reply:
Oh sorry, I think I misread the commit diff.